.travis.yml

```diff
@@ -20,14 +20,20 @@ matrix:
       env: WP_VERSION=latest
     - php: 7.2
       env: WP_VERSION=latest
+    - php: 7.2
+      env: WP_VERSION=5.3
     - php: 7.1
       env: WP_VERSION=latest
+    - php: 7.1
+      env: WP_VERSION=5.2
     - php: 7.0
       env: WP_VERSION=latest
-    - php: 5.6
-      env: WP_VERSION=latest
+    - php: 7.0
+      env: WP_VERSION=5.1
     - php: 5.6
       env: WP_VERSION=trunk
+    - php: 5.6
+      env: WP_VERSION=5.0
```

Comment:
The new class `Rest` uses some classes from Core, and has an important filter of `'rest_endpoints'` that needs to work with WP `5.0+` 

So I added more WP versions to the matrix. If this causes builds to take a lot longer, we could revert this change to `.travis.yml`.